pilot/pkg/bootstrap/options.go

```diff
@@ -155,4 +156,5 @@ func (p *PilotArgs) applyDefaults() {
 	p.KeepaliveOptions = istiokeepalive.DefaultOption()
 	p.Config.DistributionTrackingEnabled = features.EnableDistributionTracking
 	p.Config.DistributionCacheRetention = features.DistributionHistoryRetention
+	p.ServiceAddress = features.IstiodService.Get()
```

Comment:
Should we move this out of features? Actually it is not a feature to turn it off/on. We need this value. May be  define an env var here it self and use it?

Reply:
Or as a flag